src/OpenSage.Game/Utilities/TimeSpanUtility.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace OpenSage.Utilities
+{
+    public static class TimeSpanUtility
+    {
+        public static TimeSpan Max(TimeSpan a, TimeSpan b)
```

Comment:
Should this have “in”? I can’t remember how big a TimeSpan is.

Reply:
`TimeSpan` is just a wrapper around `long`, so 64 bits.

https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/TimeSpan.cs#L68